src/test/java/SmokeTest.java

```diff
@@ -0,0 +1,87 @@
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import pages.*;
+
+import java.io.IOException;
+
+public class SmokeTest extends BaseTest{
+
+    private String emailAddress = randomEmail();
+
+    private String password = randomPassword();
+    private String actualProductName;
+    private double actualPrice;
+    private int orderId;
+    private String productName = "blue jacket";
+    private String thankYouText = "Thank you for your purchase!";
+    private String accountIsCreated = "Thank you for registering with Fake Online Clothing Store.";
+
+    @BeforeTest
+    public void setUp() throws IOException {
+        driver = initializeDriver();
+    }
+
+    @Test
+    public void smokeTest() throws InterruptedException, IOException {
+
+        //open homepage and enter searched word
+        HomePage homePage = new HomePage(driver);
+        homePage.goToHomepage();
+        homePage.enterSearchedWord(productName);
+
+        SearchPage searchPage = new SearchPage(driver);
+        Assert.assertTrue(searchPage.verifySearchedText(productName));
```

Comment:
When we call an Assert method the last value we are able to pass is a String. This is used to display a message in the terminal or test report as to a reason why the assertion failed, you should add this to every assert call in this test. Do not hardcode the message in, try creating dynamic methods in the afromentioned String Utilities class that you should make. Create such methods where you are at least able to pass 1  or 2 values so that the it can output a appropriate text in case the test fails. Example: `Assert.assertTrue(searchPage.verifySearchedText(productName),Utilities.pageLoadFail("SearchPage"));`

This calls the utilities class pageLoadFail method which forms a full senate example "Failed to load page " + value, value being the "SearchPage" value we just passed to it. This means we can reuse this method for any time we assert if a webpage is loaded, the only thing we need to do is change the value we pass into it.